frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -76,31 +56,30 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
     );
   }
   return (
-    <div>
-      <span className="text-secondary">
-        <h4>Usage Breakdown</h4>
-      </span>
-      <ChartDonut
-        donutHeight={250}
-        donutWidth={110}
-        labelPosition="startAngle"
-        legendPosition="right"
-        legendData={metricLegend}
-        legendOrientation="vertical"
-        data={metricData}
-        labels={(datum) => `${datum.x}: ${datum.y.toFixed(2)}%`}
-        subTitlePosition="bottom"
-        themeColor={ChartThemeColor.multi}
-        themeVariant={ChartThemeVariant.light}
-        title={humanizeBinaryBytesWithoutB(total).string}
-        radius={53}
-        innerRadius={46}
-      />
+    <div className="nb-capacity-card__body">
+      <span className="nb-capacity-card__title">Usage Breakdown</span>
+      <div className="noobaa-capacity-card__item">
+        <ChartDonut
+          width={275}
+          height={150}
+          labelPosition="centroid"
+          legendPosition="right"
+          legendData={metricLegend}
+          legendOrientation="vertical"
+          data={metricData}
+          labels={(datum) => `${datum.x}: ${datum.y.toFixed(2)}%`}
```

Comment:
I think here we can use `humanizePercentage` in `units.js` .

Reply:
I probably have to use some thing like below
```typescript
           `${datum.x}: ${humanizePercentage(totalUsage ? datum.y.toFixed(1) : 0).string}`
```
instead of
``` typescript
          `${datum.x}: ${totalUsage ? datum.y.toFixed(1) : 0}%`
```
If you insist, will change it, but I prefer to go with a simple solution...

Reply:
its better to use `humanizePercentage` because of consistency. You are using 2 fraction digits, but rest of the console is using `humanizePercentage` which uses 1 fraction digit. And once we change that it will be consistently changed everywhere.

i think that simple
```
${datum.x}: ${humanizePercentage(datum.y).string}
```
should work

Reply:
@afreen23 had the same point... Will change it...

Reply:
Updated the PR